.circleci/config.yml

```diff
@@ -115,17 +115,17 @@ jobs:
           command: (git submodule sync && git submodule update --init --recursive) || (rm -fr .git/config .git/modules && git submodule deinit -f . && git submodule update --init --recursive)
       - restore_cache:
           keys:
-            - dev-bundle-cache-{{ checksum "meteor" }}
-            - dev-bundle-cache
+            - v1-dev-bundle-cache-{{ checksum "meteor" }}
+            - v1-dev-bundle-cache-
       - restore_cache:
           keys:
-            - other-deps-cache-{{ .Branch }}-{{ .Revision }}
-            - other-deps-cache-{{ .Branch }}
-            - other-deps-cache
+            - v1-other-deps-cache-{{ .Branch }}-{{ .Revision }}
+            - v1-other-deps-cache-{{ .Branch }}
+            - v1-other-deps-cache-
       - restore_cache:
           keys: 
-            - test-groups-v1-{{ .Branch }}
-            - test-groups-v1
+            - v1-test-groups-{{ .Branch }}
+            - v1-test-groups-
```

Comment:
@abernix @hwillson Just flagging this because invalidating the cache apparently solved the `add debugOnly and prodOnly packages` test failures. In the future, if we have any doubts about the impact of caching, we can bump these `v1-` prefixes to invalidate everything (per Circle CI recommendation [here](https://circleci.com/docs/2.0/caching/#clearing-cache)).

Possibly relevant to #9785 (cc @jamesmillerburgess). Caching can create additional complexity, but invalidating the cache seems easy enough…